setup.py

```diff
@@ -3,11 +3,15 @@
 
 
 # always prefer setuptools over distutils
-from setuptools import setup, find_packages
+import setuptools
+from numpy.distutils.core import setup, Extension
+
+# specify fortran extensions to build with numpy.f2py
+solid_ext = Extension(name='pysolid.solid', sources=['pysolid/solid.for'])
 
 setup(
     name='pysolid',
-    version='0.1.2',
+    version='0.2.0',
```

Comment:
We'll want to cut a new version for conda-forge, so I've bumped the version number here. Happy to back this out, or change it to a different number.

Reply:
Good call. Bumping to version `0.2.0` sounds good to me too.